### DISCUSSION TITLE
MPT-1327 Guru in the house?

### DISCUSSION THREAD

**Discussion by Bowlieweekender** [DSheirer/sdrtrunk#1859] [Q&A]
I've been given the control channel frequency (161.925) for a local power company running MPT-1327  Sure enough it is very strong and pumps out Aloha SYSTEM and System Parameter messages 24/7.  On a few occasions there has been actual command traffic but it says the Channel number is not known.  With just the control channel known is it possible for SDRTRUNK to discover the frequencies and or system channels that a unit is being directed to use?  Also, in the Channel Map Editor, I only know the local control channel so not sure what to put in three of the four boxes: First Channel, Last Channel, Base Frequency.  Step is 12.5KHz.  Pointers appreciated.

**Comment by DSheirer** [comment]
I created a FAQ entry here:

https://github.com/DSheirer/sdrtrunk/wiki/FAQ#how-do-i-recover-the-channel-map-for-an-mpt1327-system-if-i-only-know-the-control-channel-frequency
